### PR TITLE
chore: refactor by removing a couple variables

### PR DESCRIPTION
**Pull request by styfle**
Following up due to comment https://github.com/lovell/detect-libc/pull/19#discussion_r1266910023

See related PR:

- https://github.com/lovell/detect-libc/pull/19

### CODE REVIEW

**Comment by styfle** [comment]
@H4ad Do you have the systems you benchmarked from #19 to try this out on to make sure I didn't slow anything down?

**Comment by H4ad** [comment]
@styfle No problem, same overall performance, could be a bit slower because of `.toLowerCase`, but I don't think it's enough to slow down in this case.

**Review by H4ad — APPROVED** [review]

File: lib/detect-libc.js [hunk 1]
```diff
@@ -94,11 +82,12 @@ const familyFromCommand = (out) => {
   return null;
 };
 
-const getFamilyFromLddContent = (content) => {
-  if (content.includes(MUSL_ON_LDD)) {
+const getFamilyFromLddContent = (lddContent) => {
+  const content = lddContent.toLowerCase();
+  if (content.includes(MUSL)) {
```
Comment (lovell) [inline lib/detect-libc.js]:
```suggestion
const getFamilyFromLddContent = (content) => {
  if (content.includes(MUSL)) {
```

File: lib/detect-libc.js [hunk 2]
```diff
@@ -94,11 +82,12 @@ const familyFromCommand = (out) => {
...
     return MUSL;
   }
-  if (content.includes(GLIBC_ON_LDD)) {
+  if (content.includes(GLIBC)) {
```
Comment (lovell) [inline lib/detect-libc.js]:
```suggestion
  if (content.includes('GLIBC')) {
```

**Review by lovell — COMMENTED** [review]
Given the following:

- we're thinking about micro-optimisations
- we only need to check for the string literal `"GLIBC"` in all capitals (or the string literal `"musl"` in all lower case)
- the length of `/usr/bin/ldd` file on the latest Ubuntu is 5390 bytes

how about these changes?

File: lib/detect-libc.js [hunk 3]
```diff
@@ -95,10 +83,10 @@ const familyFromCommand = (out) => {
 };
 
 const getFamilyFromLddContent = (content) => {
-  if (content.includes(MUSL_ON_LDD)) {
+  if (content.includes(MUSL)) {
```
Comment (styfle) [inline lib/detect-libc.js]:
```suggestion
  if (content.includes('musl')) {
```

@lovell Maybe inline this one to make it consistent? I'm worried someone is going to look back years from now and think that the missing quotes is a typo.

Comment (lovell) [reply lib/detect-libc.js]: "someone... years from now" = me next week

**Review by lovell — APPROVED** [review]

File: lib/detect-libc.js (hunk 3 above)
Comment (styfle) [reply lib/detect-libc.js]: @lovell There's a [quote about this](https://twitter.com/styfle/status/1187715719846338560) 😆 

While I have your attention, do you think you could respond to https://github.com/lovell/sharp/pull/3522 when you have a moment? 🙏

**Pull request merged**